### PR TITLE
Feature/simplified action

### PR DESCRIPTION
**Pull request by nchiasson-dgi**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -59,19 +59,19 @@ runs:
...
+    - name: Copy-Paste
       shell: bash
-      if: ${{ always() && inputs.phpcpd-exclude == '' }}
+      if: always()
       continue-on-error: ${{ inputs.phpcpd-continue == 'true' }}
-      run: >
-        ${{ github.action_path }}/vendor/bin/phpcpd --suffix=module --suffix=install --suffix=inc --suffix=theme --suffix=test --suffix=profile --suffix=${{ inputs.suffix }} ${{ inputs.path }}
+      run: |
+        output=""
+        IFS=',' read -ra SUFFIX <<< ${{ inputs.suffix }}
```
Comment (Alexander-Cairns) [inline action.yml:68]:
```suggestion
        IFS=',' read -ra SUFFIX <<< "${{ inputs.suffix }}"
```

Comment (Alexander-Cairns) [reply action.yml:68]: Quote to make it read as string even if someone adds spaces

File: action.yml [hunk 2]
```diff
@@ -59,19 +59,19 @@ runs:
...
+        for i in "${SUFFIX[@]}"; do
```
Comment (Alexander-Cairns) [inline action.yml:69]: I prefer not using `i` if it's not the index in a loop.

**Review by Alexander-Cairns — CHANGES_REQUESTED** [review]

File: action.yml (hunk 2 above)
Comment (nchiasson-dgi) [reply action.yml]: Changed `i` to `suffix` to better indicate what is being used.

**Review by Alexander-Cairns — APPROVED** [review]

**Pull request merged**